test/e2e/tests/test_monitoring_schedule.py

```diff
@@ -166,34 +168,34 @@ def test_smoke(
         # as the main objective is to test for Scheduled status
         # OR
         # create the schedule with a on-going baseline job where it waits for the baselining job to complete
-        assert (
-            wait_resource_monitoring_schedule_status(
-                reference, self.STATUS_PENDING, 5, 2
-            )
-            == self.STATUS_PENDING
-        )
-        assert k8s.wait_on_condition(reference, "ACK.ResourceSynced", "False", 5, 2)
+        # assert (
```

Comment:
it's fine to remove this check if it's flaky. is it flaky?

Reply:
was flaky because of some sleeps that were added however they have been removed